packages/google_sign_in/google_sign_in/pubspec.yaml

```diff
@@ -25,7 +25,7 @@ dependencies:
   # validation, so we set a ^ constraint.
   # TODO(amirh): Revisit this (either update this part in the  design or the pub tool).
   # https://github.com/flutter/flutter/issues/46264
-  google_sign_in_web: ^0.8.2
+  google_sign_in_web: ^0.9.0
```

Comment:
In order to publish this, you'll also need to update the version of google_sign_in in this pubspec, and add an entry to the CHANGELOG file. Can you do that?

I think the `google_sign_in` package would need to become `4.4.1+1`.